clang/lib/Driver/SanitizerArgs.cpp

```diff
@@ -1160,6 +1160,10 @@ void SanitizerArgs::addArgs(const ToolChain &TC, const llvm::opt::ArgList &Args,
       CmdArgs.push_back("-asan-stack=0");
       CmdArgs.push_back("-mllvm");
       CmdArgs.push_back("-asan-globals=0");
+
+      if (!RecoverableSanitizers.empty())
+        CmdArgs.push_back(Args.MakeArgString("-fsanitize-recover=" +
+                                             toString(RecoverableSanitizers)));
```

Comment:
Could you add a corresponding test to `clang/test/Driver/sycl-device-sanitizer.cpp`?

Reply:
Done